pymongo/response.py

```diff
@@ -35,7 +35,7 @@ def __init__(
         request_id: int,
         duration: Optional[timedelta],
         from_command: bool,
-        docs: List[Mapping[str, Any]],
+        docs: List[_DocumentOut],
```

Comment:
Why did we need this change?

Reply:
At some point, docs was being a `List[_DocumentOut]` and because lists are invariant, I changed it. Would it be better if I change the `List`, to be a `Sequence`?